mls-rs/src/group/resumption.rs

```diff
@@ -186,13 +190,18 @@ impl<C: ClientConfig + Clone> ReinitClient<C> {
             extensions: self.reinit.new_group_context_extensions(),
         };
 
+        // TODO, should these come from the reinit proposal or as an arg instead?
+        let group = self.client.load_group(self.reinit.group_id())?;
+        let current_leaf_node_extensions = &group.current_user_leaf_node()?.extensions;
+
```

Comment:
There are no leaf node extensions in the reinit proposal -- only group context extensions... I'd say we don't support updating extensions in reinit and that's fine. Reinit is too complicated as it is.

Reply:
Sounds good to me. I'll remove the TODO comment then.

Reply:
Had to update this to be a parameter, Turns out load group won't work if the group has been reinited....